### PR TITLE
Update customization.md

### LINKED ISSUE #202: [Feature Request] Contextual help on selections
Provide contextual help:
```toml
[[tool.commitizen.customize.questions]]
type = "list"
choices = ["foo", "barbuz"]
foo.message = "explanation to foo"
barbuz.message = "explanation to barbuz"
```
```shell
cz commit                             
...
foo     - explanation to foo
barbuz  - explanation to barbuz
```
Help message alignment + at least two spaces before `-` (sic!)

Without it this tool is barely user friendly. Still, great tool!

### PR DESCRIPTION
**Pull request by blaggacao**
fixes #202 - also fixes other inconsistencies which contributed to my initial confusion.

### CODE REVIEW

File: docs/customization.md [hunk 1]
```diff
@@ -253,13 +254,13 @@ message = "Do you want to add body message in commit?"
...
 | `name`    | `str`  | `None`  | The key for the value answered by user. It's used in `message_template`                                                                                                                                                      |
 | `message` | `str`  | `None`  | Detail description for the question.                                                                                                                                                                                         |
-| `choices` | `list` | `None`  | (OPTIONAL) The choices when `type = choice`. It should be list of dictionaries with `name` and `value`. (e.g., `[{value = "feature", name = "feature: A new feature."}, {value = "bug fix", name = "bug fix: A bug fix."}]`) |
+| `choices` | `list` | `None`  | (OPTIONAL) The choices when `type = list`. Either use a list of values or a list of dicitionaries with `name` and `value` keys. See examples above. |
```
Comment (blaggacao) [inline docs/customization.md:264]: Maybe define the proper type set supported by upstream library in this way.

**Comment by blaggacao** [comment]
Please squash-merge with proper commit message I just did a quick one-shot modification over github ui.

**Comment by Lee-W** [comment]
Thanks for updating! I just squahed and merged it.

**Comment by blaggacao** [comment]
Super cool, I hope it's clearer now for future readers. Thx!

**Pull request merged**